
.. xexc:: datatable.exceptions.ValueError
    :src: --

    Inherits from Python :py:exc:`ValueError` and :exc:`datatable.exceptions.DtException`.
